### PR TITLE
Added skeleton code

### LINKED ISSUE #132: As an instructor, I want to have skeleton code in the create problem code editor step so I can have some baseline of what goes where
# User story
As an instructor, I want to have skeleton code in the create problem code editor step so I can have some baseline of what goes where.

we should show an example of what should go in the header, body and footer in the code editor
Add template for that first step - most programs have an int main and a header

e.g.
`
header:
using namespace std;
#include <stdio>

body:
int addTwoNums(int a, int b) {

}

footer:
int main() {
  int x, y;
  cin >> x >> y;
  cout << addTwoNums(x, y);
  return 0;
}
`

# Acceptance criteria
- [ ] There is skeleton code in the code editor (header, body, footer)

### PR DESCRIPTION
**Pull request by joshuadargan**
# Description
Adds some skeleton code for creating problems in the header, body, and footer.

# Related Issue
see #132 

# Motivation and Context
see #132

# How Has This Been Tested?
Manually on new problems and existing problems. I also ran the code to made sure that it compiles.

# Screenshots (if appropriate):
image

closes #132

### CODE REVIEW

**Comment by joshuadargan** [comment]
[default code.txt](https://github.com/edugator-cise/edugator-staff-frontend/files/7629428/default.code.txt)

**Review by AqSlz — COMMENTED** [review]
Your screenshot does not reflect the current changes (skeleton code).

File: src/pages/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx [hunk 1]
```diff
@@ -37,9 +38,103 @@ interface Errors {
...
+    else if(command == "remove")
+    {
+      std::cin >> ufid;
+      myMap.remove(ufid);
+    }  
+  }
+  */
+  return 0;
+}
+`;
```
Comment (AqSlz) [inline src/pages/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx:131]: It is nice to give them a real example but I worry that users will be annoyed when they have to delete all of this code every time want to create a new problem. Unless everyone else thinks that won't be an issue.

Comment (kapooramanpreet) [reply src/pages/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx:131]: Agree with Adrian. Body looks good to me. In the header just use:

```
//HEADER
//If students import packages or use namespaces on their own, it shouldn't cause problems
#include <iostream>
#include <vector>
using namespace std;
```


In the footer, use:
```
//FOOTER
// The main does not have to be in the footer.
// The main should remain in the footer if you don't want students to be able to see it nor change it.
int main()
{
    int x = 0, y = 0;
    cin >> x >> y;
    int result = addTwoNums(x, y);
    // You should print out whatever the expected output should be. 
    // Be careful about whitespace. Ex: only put endl if you add an endline in your expected output.
    cout << result;
    return 0;
}
```

Comment (joshuadargan) [reply src/pages/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx]: fixed. please resolve if you think it is fixed as well.

**Comment by joshuadargan** [comment]
**
image
**

**Review by kapooramanpreet — APPROVED** [review]
Looks good.

**Review by AqSlz — APPROVED** [review]

**Pull request merged**